docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -111,7 +111,7 @@ As consumers of services, we need to define what exactly we want to achieve. We
 formulate our expectations. That is why we write contracts. In other words, a contract is
 an agreement on how the API or message communication should look. Consider the following example:
 
-Assume that you want to send a request that contains the ID of a client company and the
+Assume that you want to send a request that contains the ID of a client company, and the
```

Comment:
Do not add this comma. The rule about commas before conjunction is that the part behind the conjunction must be an independent clause (capable of being a sentence by itself). That's not the case here. This conjunction creates a two-item list.